app/controllers/authentication.js

```diff
@@ -1,229 +1,238 @@
+          callback({ status: 500, msg: err.toString() });
+        } else {
+          accessToken = qs.parse(accessToken);
+          const headers = { 'User-Agent': 'Satellizer' };
 
-            callback(null, accessToken, headers);
-          }
-        });
+          callback(null, accessToken, headers);
+        }
+      });
     };
 
     /*
       Retrieve the user's github profile.
     */
-    var getProfile = function (accessToken, headers, callback) {
-      request.get({ url: userApiUrl, qs: accessToken, headers, json: true },
-        function (err, response, profile) {
-          if (err) {
-            winston.info('Github auth: getProfile error');
-            callback({ status: 500, msg: err.toString() });
-          } else if (!err && response.statusCode !== 200) {
-            winston.info('Github auth: bad profile response');
-            callback({ status: 409, msg: 'Did not get github profile.' });
-          } else if (!profile.email) {
-            winston.info('Github auth: no email error');
-            callback({ status: 409, msg: 'Did not get email address.' });
-          } else {
-            callback(null, accessToken, headers, profile);
-          }
-        });
+    const getProfile = (accessToken, headers, callback) => {
+      request.get({ url: userApiUrl, qs: accessToken, headers, json: true }, (err, response, profile) => {
+        if (err) {
+          winston.info('Github auth: getProfile error');
+          callback({ status: 500, msg: err.toString() });
+        } else if (!err && response.statusCode !== 200) {
+          winston.info('Github auth: bad profile response');
+          callback({ status: 409, msg: 'Did not get github profile.' });
+        } else if (!profile.email) {
+          winston.info('Github auth: no email error');
+          callback({ status: 409, msg: 'Did not get email address.' });
+        } else {
+          callback(null, accessToken, headers, profile);
+        }
+      });
     };
 
     /*
       Ensure the user belongs to the required organization.
     */
-    var checkOrganization = function (accessToken, headers, profile, callback) {
-      request.get({ url: userApiUrl + '/orgs', qs: accessToken, headers, json: true },
-        function (err, response) {
-          if (err) {
-            winston.info('Github auth: checkOrganization error');
-            callback({ status: 500, msg: err.toString() });
-          } else {
-            var belongsOrg = _.find(response.body, { login: nconf.get('github').organization });
+    const checkOrganization = (accessToken, headers, profile, callback) => {
+      request.get({ url: `${userApiUrl}/orgs`, qs: accessToken, headers, json: true }, (err, response) => {
+        if (err) {
+          winston.info('Github auth: checkOrganization error');
+          callback({ status: 500, msg: err.toString() });
+        } else {
+          const belongsOrg = _.find(response.body, { login: nconf.get('github').organization });
```

Comment:
move `nconf.get..organization` to top of the file as const.

+
add env.example.json:
```
  "GITHUB_ORG": "ORGANIZATION",
  "GITHUB_ADMINTEAM": "ADMIN-TEAM",
```